examples/30_extended/fetch_runtimes_tutorial.py

```diff
@@ -389,9 +396,10 @@ def get_incumbent_trace(trace):
 ################################################################################
 # Summmary
 # *********
-# OpenML records model runtimes for the CPU-clock and the wall-clock times. The above
-# examples illustrated how these recorded runtimes can be extracted, especially when
-# using a scikit-learn model and under parallel setups too. To summarize, OpenML measures the:
+# OpenML records model runtimes for the CPU-clock and the wall-clock times for
```

Comment:
There was a comment which got resolved which read like

> Hm, should we explicitly say "the scikit-learn extension"? Everything we're writing here is exclusive about the scikit-learn extension, so it could be confusing otherwise.

and I cannot add text there any more, but I don't think it's resolved yet. It still says OpenML everywhere, though this is only about the scikit-learn extension.